pele_platform/Utilities/Helpers/constraints/alpha_constraints.py

```diff
@@ -0,0 +1,228 @@
+from collections import defaultdict
+from prody.proteins.pdbfile import parsePDB
+import pele_platform.constants.constants as constants
+import pele_platform.Utilities.Helpers.map_atoms as map_atoms
+from PPP.checks_module import CheckforGaps
+
+
+terminal_constr_spring = 5
+backbone_constr_spring = 0.5
+
+CONSTR_ATOM = """{{ "type": "constrainAtomToPosition", "springConstant": {0}, "equilibriumDistance": 0.0, "constrainThisAtom": "{1}:{2}:{3}" }},"""
+CONSTR_DIST = """{{ "type": "constrainAtomsDistance", "springConstant": {}, "equilibriumDistance": {}, "constrainThisAtom": "{}:{}:{}", "toThisOtherAtom": "{}:{}:{}" }},"""
+CONSTR_CALPHA = """{{ "type": "constrainAtomToPosition", "springConstant": {2}, "equilibriumDistance": 0.0, "constrainThisAtom": "{0}:{1}:_CA_" }},"""
+
+
+class AlphaConstraints(object):
+    """
+    Class to parse alpha carbons in the PDB file and constrain them according to the user-defined settings.
+    """
+    def __init__(self, pdb, interval, backbone_spring, terminal_spring):
+        self.pdb = pdb
+        self.interval = interval
+        self.backbone_spring_constant = backbone_spring
+        self.terminal_spring_constant = terminal_spring
+        self.residues = self.get_all_residues()
+        self.interval_residues = self._apply_interval()
+        self.gaps = self.find_gaps()
+
+    def build_constraints(self):
+        """
+        Run the whole carbon alpha constraints pipeline, including terminals, gaps and interval backbone.
+
+        Returns:
```

Comment:
I was wrong when confirming to you the docstrings style we should use is Google's. The one I prefer is the following: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Sorry about that!! 🙏 
We do need to establish all these best practices in a document so people does not get confused (like me 😛)

Reply:
So
```python
"""
Run the whole carbon alpha constraints pipeline, including terminals, gaps and interval backbone.

Returns
--------
constraints : list[str]
    The list of string constraints ready to be injected into the PELE configuration file
"""
```